static/js/popovers.js

```diff
@@ -510,6 +510,23 @@ exports.register_click_handlers = function () {
         e.stopPropagation();
         e.preventDefault();
     });
+    $('body').on('click', '.view_edit_history', function (e) {
+        var msgid = $(e.currentTarget).data('msgid');
+        var row = current_msg_list.get_row(msgid);
+        var message = current_msg_list.get(rows.id(row));
+
+        popovers.hide_actions_popover();
+        $('#message-edit-history').modal("show");
+        // _.each(message.edit_history, function (old_message) {
```

Comment:
@neerajwahi any suggestion on how to convert timestamp from `1467987508` to correct time. This commented code gives incorrect date and time.